src/js/angular/repositories/controllers.js

```diff
@@ -469,12 +485,24 @@ function AddRepositoryCtrl($scope, toastr, $repositories, $location, Upload, isE
         });
     };
 
-    if ($scope.isEnterprise) {
-        $scope.getConfig('worker');
-    } else if ($scope.isFreeEdition) {
-        $scope.getConfig('free');
+    $scope.isRepositoryTypeValid = function () {
```

Comment:
Is this used somewhere in templates? If not, then you don't need to clutter the scope with unneeded functions

Reply:
No it's not used, I will take it out of the scope